.envrc

```diff
@@ -0,0 +1,14 @@
+# direnv Configuration File
```

Comment:
We are starting to use direnv, to manage enviroment/interpreter, for now on?

Reply:
We do not discuss this, I am justing using it because I am familiar with and we do not have any clear and official way to manage it